packages/@aws-cdk/aws-rds/lib/cluster-ref.ts

```diff
@@ -14,6 +14,13 @@ export interface IDatabaseCluster extends IResource, ec2.IConnectable, secretsma
    */
   readonly clusterIdentifier: string;
 
+  /**
+   * The immutable identifier for the cluster; for example: cluster-ABCD1234EFGH5678IJKL90MNOP.
+   *
+   * This AWS Region-unique identifier is used in things like IAM authentication policies.
```

Comment:
```suggestion
   * This AWS Region-unique identifier is used to grant access to the cluster.
   *
   * @see https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/UsingWithRDS.IAMDBAuth.IAMPolicy.html
```

Reply:
Hey @comcalvi, thanks for the review 😄 

Unfortunately the ARN isn't what we need here. The cluster identifier and resource identifier are different.

**RDS Cluster ARN:** arn:aws:rds:_region_:_account_:_clusterIdentifier_
**RDS Cluster IAM Auth Access ARN:** arn:aws:rds:_region_:_account_:dbuser:_clusterResourceIdentifier_/_databaseUser_

I was following the existing examples of [docdb](https://github.com/aws/aws-cdk/blob/74318c7d22bfc00de9e005f68a0a6aaa58c7db39/packages/%40aws-cdk/aws-docdb/lib/cluster.ts#L324-L344) and [neptune](https://github.com/aws/aws-cdk/blob/74318c7d22bfc00de9e005f68a0a6aaa58c7db39/packages/%40aws-cdk/aws-neptune/lib/cluster.ts#L243-L252) that expose both the cluster identifier and cluster resource identifiers as properties.